ibm/resource_ibm_cis_range_app_test.go

```diff
@@ -0,0 +1,244 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccIBMCisRangeApp_Basic(t *testing.T) {
+	var app string
+	name := "ibm_cis_range_app.app"
+	originDNS := fmt.Sprintf("test-lb.%s", cisDomainStatic)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckCis(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCisRangeAppConfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCisRangeAppExists(name, &app),
+					resource.TestCheckResourceAttr(name, "origin_direct.#", "1"),
```

Comment:
Can more feature checks be added to testcase?

Reply:
added